tools/test_reference_check.py

```
@@ -76,14 +77,48 @@ def main():
+    # repo, compute the sha at test time, and the legs measure the SAME PROPERTY with nothing
+    # borrowed.
+    _tmp = tempfile.mkdtemp()
+    _repo = os.path.join(_tmp, "sibling-repo")
+    os.makedirs(os.path.join(_repo, "docs"))
+    _doc = os.path.join(_repo, "docs", "x.md")
+    with open(_doc, "w", encoding="utf-8") as fh:
+        fh.write("authoritative for things\n")
+    for _cmd in (["git", "init", "-q"], ["git", "add", "-A"],
+                 ["git", "-c", "user.email=t@t", "-c", "user.name=t",
+                  "commit", "-q", "-m", "seed"]):
+        subprocess.run(_cmd, cwd=_repo, capture_output=True)
```

Ruff (0.16.3)

[error] `subprocess` call: check for execution of untrusted input

(S603)